North-Seattle-College/ad440-winter2022-thursday-repo
# Import data in dynamodb

issue #4  

**Task Description** 
 
Develop a script to import the Floop data into Dynamodb and the output is the table format of floop-conv-data.py  

**Prerequisites**

In terminal run pip install boto3 

Downloaded the Floop data file in s3 bucket in AWS console and saved it as floop-conv-data.json in my project directory folder. 

**To Test** 

Run the script python floop-data-import.py it takes 5-8 minutes. 
Go to the dynamodb and Run the table  
The result should be like this:


![Screenshot (34)](https://user-images.githubusercontent.com/38993144/151755791-18bc5a27-faef-4ba5-bd1b-0385cbdee6e3.png)

<html>
<body>


Activities | Time Spent
-- | --
Researching about dynamodb and connecting to AWS dynamodb | 6 hours
Research how to create table on AWS dynamodb | 2 hours
Dynamodb set up and creating table | 3 hours
Writing the script | 4 hours
Deployment | 1 hour
Total hours spent | 16 hours


</body>
</html>
